weaver/samples/besu/besu-cli/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "strict": false,
     "target": "es5",
     "declaration": true,
+    // "rootDir": "./",
```

Comment:
Remove

Reply:
I'll remove this, actually there's a small issue, there's a global `node_modules` folder when we clone the `cacti` repo, and modules from that conflicts with weaver node packages, this `rootDir` kinds of fixes it but sometimes create other issues. So first I added it and then commented it out.

Reply:
OK, remove it for now and open an issue to track this.